test/parallel_test/test_comm.py

```diff
@@ -12,6 +12,7 @@
 def test_comm(lib, procs):
     comm = get_cart(procs)
     lib.set_comm(comm)
+    lib.init_quda()
```

Comment:
Not clear why `lib.init_quda` needs to be called after `set_comm`.
It shouldn't restart by it's own if it gets finalized?